apps/remix-ide/src/app/ui/landing-page/landing-page.js

```diff
@@ -193,7 +194,7 @@ export class LandingPage extends ViewPlugin {
           href="https://twitter.com/EthereumRemix"
         >
         </a>
-        <script src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+        <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```

Comment:
why do you need to load the same script twice?

Reply:
because I couln't change data-theme

Reply:
ah you are right

Reply:
or not. I use update of yo-yo